.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "\U0001F41B Bug Report"
+about: "Please report issues related to the TypeScript and HTML API of Vaadin components here."
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue risks taking an infinite amount of time to triage. This info allows the maintainers team to diagnose and fix your issue as quickly as possible.
+
+  Before opening a new issue, please search existing issues: https://github.com/vaadin/web-components/issues.
+-->
+
+### Description
+
+Describe the issue that you're seeing.
+
+<!-- Example: The `vaadin-grid` element scrolls when applying filter. -->
+
+### Expected outcome
+
+What should happen?
+
+<!-- Example: No scrolling should happen. -->
+
+### Actual outcome
+
+What happened.
+
+<!-- Example: Grid content occasionally gets scrolled. -->
+
+### Live Demo
+
+A screenshot or video with issue reproduced.
+
+### Minimal reproducible example
+
+**Please** provide a code snippet with issue reproduction, this makes your issue _much_ easier to diagnose.
```

Comment:
```suggestion
**Please** provide a demo project or code snippet that reproduces the issue, this makes your issue _much_ easier to diagnose.
```
I would also like to be more forceful here by stating that issues without reproduction are unlikely to be worked on. The current wording could easily lead to a mindset of "Not my problem if it takes you longer to diagnose", while the reality is that we often can not do anything without a reproduction. Could also have a quick check with Rolf for a reasonable wording of this.